lib/logpipe.h

```diff
@@ -363,6 +366,12 @@ log_pipe_append(LogPipe *s, LogPipe *next)
   s->pipe_next = next;
 }
 
+void
+log_pipe_set_persist_id(LogPipe *self, const gchar *persist_id);
+
+const gchar *
+log_pipe_get_persist_id(const LogPipe *self);
```

Comment:
i'd prefer log_pipe_get_persist_name, since the returned value is usually prefix+persist_id (e.g. afsocket_dd.persistid)